dbfit-java/informix/src/test/java/dbfit/environment/InformixBigDecimalTransformerTest.java

```diff
@@ -0,0 +1,45 @@
+package dbfit.environment;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.math.BigDecimal;
+import java.sql.SQLException;
+
+import dbfit.util.NormalisedBigDecimal;
+
+public class InformixBigDecimalTransformerTest {
+
+    @Test
+    public void transformsTest() {
+        InformixBigDecimalTransformer bdt = new InformixBigDecimalTransformer();
+        BigDecimal inBigDec = new BigDecimal("123.45");
+        NormalisedBigDecimal nbd = new NormalisedBigDecimal(inBigDec);
+        BigDecimal outBigDec = null;
+        try {
+            outBigDec = (BigDecimal) bdt.transform(nbd);
+        } catch (SQLException e) {
+            fail("InformixBigDecimalTransformer threw SQLException transforming NormalisedBigDecimal input");
+        }
+        assertEquals(outBigDec, inBigDec);
+    }
+
+    @Test
+    public void rejectsInputTest() {
+        InformixBigDecimalTransformer bdt = new InformixBigDecimalTransformer();
+        BigDecimal inBigDec = new BigDecimal("123.45");
+        boolean caught = false;
+        String expectedMsg = "InformixBigDecimalTransformer cannot transform objects of type java.math.BigDecimal";
```

Comment:
> So perhaps InformixNormalisedBigDecimalTransformer is a better name. Hmmmm. Then again may be even NormalisedBigDecimalTransformer and make it DB-type agnostic. Any view?

I'm trying to put together the bits around the the current situation:

* In a transformer we have a `source` and a `destination` object.
* So it may make sense in the name to emphasize on any of them or maybe rather on both.
* In case of in-going transformations:
  - we call them `normaliser`s
  - and the name is usually focusing on the `source`. (E.g. for clob normaliser the destination is a String).
* Now in case of `InformixBigDecimalTransformer`: 
  - we have this generic `*Transformer` name which doesn't give a clear idea between which worlds are we transforming (is it DbFit -> JDBC or something else).
  - and we're a bit confused whether we should emphasize on `source` or `destination` in the name.

So:
 * Should we somehow indicate the `direction` of the transformer in the class name - e.g. `*OutTransformer` or `*OutgoingTransformer` or `<Object>ToJDBCTransformer`...?
 * Do we focus on the `source` or on the `destination` in the name (or both). If it's still only single one - maybe the `source` is more relevant.

And in our particular case for Informix BigDecimal - I guess we can skip the check and allow it to work for any `BigDecimal` as a source (not just normalised one).